templates/shared/forms/interactive/managed-kubernetes.html

```diff
@@ -151,7 +34,7 @@ <h3 class="p-heading--five">How should we get in touch?</h3>
                 <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox" />
                 <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</label>
               </li>
-              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</li>
+              <li class="p-list__item">In submitting this form, I confirm that I have read and agree to Canonical's<a href="/legal/terms-and-policies/privacy-policy"> Privacy Policy</a></li>
```

Comment:
Probably should be a space before the link here

Reply:
Fixed👍🏻